modules/fidesModule/persistence/threat_intelligence.py

```diff
@@ -0,0 +1,12 @@
+from typing import Optional
```

Comment:
what's the difference between this dir **modules/fidesModule/persistence** and **modules/fidesModule/persistance** ?? is it possible to use more descriptive names if they're used for different purposes?

Reply:
They existed because of historical/developmental reasons, they should be "merged" into one now.